specification/dataprotection/resource-manager/Microsoft.DataProtection/stable/2023-01-01/dataprotection.json

```diff
@@ -0,0 +1,7082 @@
+      "properties": {
+        "eTag": {
+          "description": "Optional ETag.",
+          "type": "string"
+        },
+        "id": {
+          "description": "Resource Id represents the complete path to the resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "location": {
+          "description": "Resource location.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Resource name associated with the resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "tags": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags.",
+          "type": "object"
+        },
+        "type": {
+          "description": "Resource type represents the complete path of the form Namespace/ResourceType/ResourceType/...",
+          "readOnly": true,
+          "type": "string"
+        },
+        "systemData": {
+          "$ref": "#/definitions/systemData"
+        }
+      },
+      "type": "object",
+      "x-ms-azure-resource": true
+    },
+    "DppTrackedResource": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/DppBaseTrackedResource"
+        }
+      ],
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/DppIdentityDetails",
+          "description": "Input Managed Identity Details"
+        }
+      },
+      "type": "object",
+      "x-ms-azure-resource": true
+    },
+    "ImmutabilitySettings": {
+      "description": "Immutability Settings at vault level",
+      "type": "object",
+      "properties": {
+        "state": {
+          "description": "Immutability state",
+          "enum": [
+            "Disabled",
+            "Unlocked",
+            "Locked"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ImmutabilityState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "DppProxyResource": {
+      "properties": {
+        "id": {
+          "description": "Proxy Resource Id represents the complete path to the resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "name": {
+          "description": "Proxy Resource name associated with the resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "type": {
+          "description": "Proxy Resource type represents the complete path of the form Namespace/ResourceType/ResourceType/...",
+          "readOnly": true,
+          "type": "string"
+        },
+        "tags": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Proxy Resource tags.",
+          "type": "object"
+        },
+        "systemData": {
+          "$ref": "#/definitions/systemData"
+        }
+      },
+      "type": "object",
+      "x-ms-azure-resource": true
+    },
+    "systemData": {
```

Comment:
Please can you reference systemData from common-types/v3 etc. instead of declaring it yourself

Reply:
I've addressed and referred it from common types.